input/fsh/PatientSEVendorLite.fsh

```diff
@@ -6,6 +6,9 @@ Description: "Simplified patient for identification for communication between sy
 * identifier[personnummer] ..1
 * identifier[samordningsnummer] ..1
 * identifier[nationelltReservnummer] ..1
+* identifier[personnummer].value 1..
```

Comment:
Are those not must-supports? Don't we need to add 'MS' at the end?

Reply:
The "parent" element (each identifier slice) is MS. I suppose that should be interpreted as all of its children should then also be MS, or what do you say @peturv ?